sentences/fi/light_HassTurnOff.yaml

```diff
@@ -1,8 +1,13 @@
-language: fi
+language: "fi"
 intents:
   HassTurnOff:
     data:
-      - sentences: []
+      - sentences:
+          - "<päälle> {area} <valo> <pois>"
```

Comment:
So, one possible sentence would be:
"laita päälle olohuone valo pois päältä"
Roughly translated:
"turn on livingroom light turn off"